languages/envato-market.pot

```diff
@@ -1,34 +1,40 @@
-# Copyright (C) 2022 Envato
+# Copyright (C) 2023 Envato
 # This file is distributed under the same license as the Envato Market package.
 msgid ""
 msgstr ""
-"Project-Id-Version: Envato Market 2.0.7\n"
+"Project-Id-Version: Envato Market 2.0.8\n"
 "Report-Msgid-Bugs-To: Envato Support Team <support@envato.com>\n"
-"POT-Creation-Date: 2022-05-10 22:55:10+00:00\n"
+"POT-Creation-Date: 2023-01-17 03:39:23+00:00\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"PO-Revision-Date: 2022-MO-DA HO:MI+ZONE\n"
+"PO-Revision-Date: 2023-MO-DA HO:MI+ZONE\n"
 "Last-Translator: Envato Support Team <support@envato.com>\n"
 "Language-Team: Envato Support Team <support@envato.com>\n"
 "X-Generator: grunt-wp-i18n 1.0.3\n"
 
-#: api-test/inc/EnvatoDebug.php:57 api-test/inc/EnvatoDebug.php:58
-msgid "Envato Debug"
-msgstr ""
-
-#: envato-market.php:99
+#: docs/dist/envato-market/envato-market.php:99 envato-market.php:99
 msgid ""
 "The Envato Market plugin requires PHP version 5.4+, plugin is currently NOT "
 "ACTIVE. Please contact the hosting provider to upgrade the version of PHP."
 msgstr ""
 
-#: envato-market.php:118
+#: docs/dist/envato-market/envato-market.php:118 envato-market.php:118
 msgid ""
 "Envato Market plugin is not installed correctly. Please delete this plugin "
 "and get the correct zip file from %s."
 msgstr ""
 
+#: docs/dist/envato-market/inc/admin/class-envato-market-admin.php:78
+#: docs/dist/envato-market/inc/admin/class-envato-market-admin.php:88
+#: docs/dist/envato-market/inc/class-envato-market-api.php:80
+#: docs/dist/envato-market/inc/class-envato-market-api.php:90
+#: docs/dist/envato-market/inc/class-envato-market-github.php:87
+#: docs/dist/envato-market/inc/class-envato-market-github.php:97
+#: docs/dist/envato-market/inc/class-envato-market-items.php:100
+#: docs/dist/envato-market/inc/class-envato-market-items.php:110
+#: docs/dist/envato-market/inc/class-envato-market.php:149
+#: docs/dist/envato-market/inc/class-envato-market.php:159
```

Comment:
There are lots of new inclusions from `docs/dist/` in here, are they meant to be included in this file?

Reply:
Ah never mind, you've already removed them 😄

Reply:
Yes this was a mistake, I removed those `exclude` options trying to get the updated `makepot` tool working. Forgot to bring them back.

They're back now and those entries shouldn't make it into the potfile any more: https://github.com/envato/wp-envato-market/pull/282/commits/5a60d80be9596a3a9f516d105689865b9a6f8585